.eslintrc.js

```diff
@@ -331,6 +331,7 @@ module.exports = {
 				'**/*.@(android|ios|native).js',
 				'packages/react-native-*/**/*.js',
 				'test/native/**/*.js',
+				'test/e2e/**/*.[tj]s',
```

Comment:
No need to exclude TS files, since we don't enable those rules for TS files in the first place (see a few lines above, L329).

```suggestion
				'test/e2e/**/*.ts',
```

On a side note, we might want to enable all these for TS files, and update all exclusions to exclude TS files in a separate PR.

Reply:
I copied to glob that the Playwright eslint rules target :) It should be more future-proof this way.

> On a side note, we might want to enable all these for TS files, and update all exclusions to exclude TS files in a separate PR.

Probably a good idea. The components package has the tests in TS files.

Reply:
Cool! I'm happy to follow up and enable it for TS test files once you merge this one 👍